js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -1,9 +1,11 @@
 import {Box, Checkbox, NonIdealState, SplitPanelContainer} from '@dagster-io/ui';
 import flatMap from 'lodash/flatMap';
+import pickBy from 'lodash/pickBy';
 import uniq from 'lodash/uniq';
 import uniqBy from 'lodash/uniqBy';
 import without from 'lodash/without';
 import React from 'react';
+import {useHistory} from 'react-router';
```

Comment:
`react-router-dom` -- I thought we were linting against `react-router` imports but I must have inadvertently lost it during the recent eslint config changes.

Reply:
Hmm looks like this isn't consistent throughout the project, will fix these in a follow-up